CONTRIBUTING.md

```diff
@@ -270,45 +270,45 @@ VS Code.
 rush dogfood
 ```
 
-This will globally install the @cadl-lang/compiler package, putting your
-build of `cadl` on PATH, and install the VS Code extension if VS Code
+This will globally install the @typespec/compiler package, putting your
+build of `typespec` on PATH, and install the VS Code extension if VS Code
 is installed.
 
 Note the important difference between this and the steps to run and
 debug the VS Code extension above: the `dogfood` command installs the
-Cadl extension with your changes in regular, non-experimental instance
+TypeSpec extension with your changes in regular, non-experimental instance
 of VS Code, meaning you will have it always, and not only when running
-the debug steps above. This is exactly like using `cadl vscode install`,
+the debug steps above. This is exactly like using `typespec vscode install`,
 only instead of downloading the latest release, it uses a build with your
 changes applied.
 
 There is no automatic `dogfood` process for installing the VS
-extension non-experimentally, but if you build the cadl-vs project from
+extension non-experimentally, but if you build the typespec-vs project from
 the command line following the steps above, or build its Release
 configuration in Visual Studio, then you can install it by
-double-clicking on packages/cadl-vs/Microsoft.Cadl.VisualStudio.vsix
+double-clicking on packages/typespec-vs/Microsoft.TypeSpec.VisualStudio.vsix
 that gets produced.
 
 # Pull request
 
-## Trigger Cadl Playground Try It build
+## Trigger TypeSpec Playground Try It build
 
 For contributors of the repo the build will trigger automatically but for other's forks it will need a manual trigger from a contributor.
-As a contributor you can run the following command to trigger the build and create a cadl playground link for this PR.
+As a contributor you can run the following command to trigger the build and create a typespec playground link for this PR.
 
 ```
-/azp run Cadl Pull Request Try It
+/azp run TypeSpec Pull Request Try It
```

Comment:
Let's ensure we have a tracking issue somewhere for updating this!

Reply:
Fixed.